src/tests.builds

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildNet462AndNewer Condition="'$(TargetGroup)' == 'netfx' AND '$(BuildNet462AndNewer)' == ''">false</BuildNet462AndNewer>
```

Comment:
When building build-tests.cmd -framework:netfx the tests where built against 462 and 47 (not only net461) which we don't want to do when building tests, but we do when building ref and src. I added this property because bin placing is not fully correct yet for 462 and 47 so when doing build-tests.cmd -framework:netfx tests that have any of this build configurations and uses something that is not in the targetingpack, the build will fail because we are not binplacing for those targetgroups what is produced in box.

This is supposed to exclude test projects: https://github.com/dotnet/corefx/blob/master/dir.props#L112, when building them throug msbuild it works fine because when setting <IsTestProject>true</IsTestProject> the project has /tests/ on the path. However when doing it through build-tests.cmd -framework:netfx the guy that includes dir.props is tests.builds and it doesn't have any /tests/ on the path as it is on the root, so IsTestsProject will be false and it will build tests against net462 and net47 as well. 

cc: @weshaggard @ericstj